test/spec/modules/kubientBidAdapter_spec.js

```diff
@@ -175,6 +182,94 @@ describe('KubientAdapter', function () {
       });
     }
   });
+  describe('buildRequestBanner', function () {
```

Comment:
test failures are coming from here (if you remove this describe block, they pass). Reason is `config` is global state and here it's being manipulated outside of the test lifecycle. This needs to be refactored so that calls to `config.setConfig`, `config.resetConfig` etc happen inside `before` / `after` / `beforeEach` ...